src/pages/GetHelp.vue

```diff
@@ -3,172 +3,213 @@
     <div class="wrapper">
       <div v-if="error" class="error">{{ error }}</div>
       <h1 class="q-px-lg uppercase">{{ $t('getHelp') }}</h1>
+      <q-form @submit="send()">
+        <q-stepper
+          v-model="step"
+          ref="stepper"
+          animated
+          alternative-labels
+          active-color="secondary"
+          inactive-color="primary"
+        >
+          <q-step
+            :name="1"
+            :title="$t('yourMessage')"
+            label="1"
+            :done="step > 1"
+          >
+            <h3>{{ $t('questionHelp') }}</h3>
+            <div class="row justify-center items-center">
+              <q-select
+                id="categories"
+                class="form-input q-pr-md col-xs-12 col-md-3"
+                filled
+                v-model="category"
+                :options="categories"
+                :label="$t('category')"
+                option-value="_id"
+                option-label="name"
+                map-options
+                bg-color="accent"
+              ></q-select>
+              <q-input
+                filled
+                name="title"
+                class="form-input q-pr-md col-xs-12 col-md-3"
+                :label="$t('title')"
+                v-model="title"
+                bg-color="accent"
+              />
+              <q-input
+                class="q-pr-sm form-input-date form-input col-xs-12 col-md-3"
+                filled
+                name="startdate"
+                v-model="startdate"
+                :rules="['date', 'time']"
+                :label="$t('startdate')"
+                bg-color="accent"
+              >
+                <template v-slot:append>
+                  <q-icon name="event" class="cursor-pointer">
+                    <q-popup-proxy
+                      ref="qStartDateProxy"
+                      transition-show="scale"
+                      transition-hide="scale"
+                    >
+                      <div class="q-gutter-md row">
+                        <q-date
+                          mask="DD/MM/YYYY HH:mm"
+                          v-model="startdate"
+                          :rules="['date']"
+                          color="secondary"
+                          :options="fromToday"
+                        />
+                        <q-time
+                          v-model="startdate"
+                          mask="DD/MM/YYYY HH:mm"
+                          :rules="['time']"
+                          color="secondary"
+                          @input="() => $refs.qStartDateProxy.hide()"
```

Comment:
you can make this point free: '$refs.qStartDateProxy.hide' instead of '() => $refs.qStartDateProxy.hide()'